docs/src/api/config-yaml.md

```diff
@@ -0,0 +1,148 @@
+	repo: https://github.com/example/my-rust-sdk
+	branch: main
+```
+In this case, the `ubrn checkout` command will clone the given repo with the branch/ref into the `rust_modules` directory of the project.
+
+If run a second time, no overwriting will occur.
+
+The `manifest-path` is the path relative to the root of the Rust workspace directory. In this case, the manifest is expected to be, relative to your React Native library project: `./rust_modules/my-rust-sdk/crates/my-api/Cargo.tml`.
+
+```yaml
+crate:
+	directory: ./rust
+	manifest-path: crates/my-api/Cargo.toml
+```
+In this case, the `./rust` directory tells `ubrn` where the Rust workspace is, relative to your React Native library project. The `manifest-path` is the relative path from the workspace file to the crate which will be used to build bindings.
+
+## `bindings`
+
+This section governs the generation of the bindings— the nitty-gritty of the Rust API translated into Typescript. This is mostly the location on disk of where these files will end up, but also has a second configuration file.
+
+```yaml
+bindings:
+	cpp: cpp/bindings
+	ts: ts/bindings
+	uniffiToml: ./uniffi.toml
+```
+The [`uniffi.toml` file](uniffi-toml.md) configures custom types, to further customize the conversion into Typescript data-types.
+
+If missing, the defaults will be used:
+```
+bindings:
+	cpp: cpp/generated
+	ts: ts/generated
+```
+## `android`
+
+This is to configure the build steps for the Rust, the bindings, and the turbo-module code for Android.
+
+This section can be completely missed out, as sensible defaults are provided, but can also be customized:
+
+```yaml
+android:
+	directory: ./android
+	cargoExtras: []
+	targets:
+	- aarch64-linux-android
+	- armv7-linux-androideabi
+	- i686-linux-android
+	- x86_64-linux-android
+	apiLevel: 21
+	jniLibs: src/main/jniLibs
+	packageName: <DERIVED FROM package.json>
+```
+
+The `directory` is the location of the Android project, relative to the root of the React Native library project.
+
+`targets` is a list of targets to build for. The Rust source code is built once per target.
+
+`cargoExtras` is a list of extra arguments passed directly to the `cargo build` command.
+
+`apiLevel` is the minimum API level to target: this is passed to the `cargo ndk` command as a `--platform` argument.
+
+```admonish tip
+Reducing the number of targets to build for will speed up the edit-compile-run cycle.
+```
+
+`packageName` is the name of the Android package that Codegen used to generate the TurboModule. This is derived from the `package.json` file, and can almost always be left.
+
+To customize the `packageName`, you should edit or add the entry at the path `codegenConfig`/`android`/`javaPackageName` in `package.json`.
+
+## `ios`
+
+This is to configure the build steps for the Rust, the bindings, and the turbo-module code for iOS.
+
+This section can be completely missed out, as sensible defaults are provided, but can also be customized:
```

Comment:
grammar nit: completely missed out feels strange, maybe 'completely omitted'.  Don't feel strongly, if this is a british vs american english thing.